refinery/data_set_manager/utils.py

```diff
@@ -806,10 +800,17 @@ def format_solr_response(solr_response):
         solr_response_json = json.loads(solr_response)
     except TypeError:
         return "Error loading json."
+    assert solr_response_json['responseHeader']['status'] == 0, \
```

Comment:
@scottx611x : I would like to leave this, actually: If solr is returning error responses to us, then we really do have a problem, and letting it bubble up and manifest as a 500 is appropriate: It's not something we can or should recover from.

This assertion isn't necessary for my code, but it will make it easier for any developer in the future by pointing them in the right direction.